spec/components/general_spec.rb

```diff
@@ -14,7 +14,11 @@
 end
 
 describe Minfraud::Components::Report::Transaction do
+  before do
+    Minfraud.configure { |c| c.enable_validation = false }
+  end
   describe '#initialize' do
+    Minfraud.configure { |c| c.enable_validation = false }
```

Comment:
Why are we doing this twice?

Reply:
I'm not sure that it is getting done twice. I added it because without it the following test failure occurs:
```
Failures:

  1) Minfraud::Components::Report::Transaction#initialize 
     Failure/Error: raise ArgumentError, 'At least one of the following is required: ip_address, minfraud_id, maxmind_id, transaction_id.'
     
     ArgumentError:
       At least one of the following is required: ip_address, minfraud_id, maxmind_id, transaction_id.
     # ./lib/minfraud/components/report/transaction.rb:101:in `validate'
     # ./lib/minfraud/components/report/transaction.rb:85:in `initialize'
     # ./spec/components/general_spec.rb:19:in `block (3 levels) in <top (required)>'
     # /var/lib/gems/2.7.0/gems/webmock-3.14.0/lib/webmock/rspec.rb:37:in `block (2 levels) in <top (required)>'

  2) Minfraud::Components::Report::Transaction#initialize 
     Failure/Error: raise ArgumentError, 'At least one of the following is required: ip_address, minfraud_id, maxmind_id, transaction_id.'
     
     ArgumentError:
       At least one of the following is required: ip_address, minfraud_id, maxmind_id, transaction_id.
     # ./lib/minfraud/components/report/transaction.rb:101:in `validate'
     # ./lib/minfraud/components/report/transaction.rb:85:in `initialize'
     # ./spec/components/general_spec.rb:20:in `block (3 levels) in <top (required)>'
     # /var/lib/gems/2.7.0/gems/webmock-3.14.0/lib/webmock/rspec.rb:37:in `block (2 levels) in <top (required)>'

Finished in 0.14648 seconds (files took 0.52169 seconds to load)
137 examples, 2 failures

Failed examples:

rspec ./spec/components/general_spec.rb:19 # Minfraud::Components::Report::Transaction#initialize 
rspec ./spec/components/general_spec.rb:20 # Minfraud::Components::Report::Transaction#initialize 

/usr/bin/ruby2.7 -I/var/lib/gems/2.7.0/gems/rspec-core-3.10.1/lib:/var/lib/gems/2.7.0/gems/rspec-support-3.10.2/lib /var/lib/gems/2.7.0/gems/rspec-core-3.10.1/exe/rspec --pattern spec/\*\*\{,/\*/\*\*\}/\*_spec.rb failed
```

Reply:
Could we remove the other one in that case? Is it doing anything?

Reply:
Oh nevermind, I couldn't see the other instance I added in the diff shown with your comment 😬